Makefile

```diff
@@ -125,6 +125,9 @@ e2e-test-erc20-hub-to-rollapp-without-register:  clean-e2e
 e2e-test-rollapp-upgrade-non-state-breaking-evm: clean-e2e
 	cd tests && go test -timeout=25m -race -v -run TestRollappUpgradeNonStateBreaking_EVM .
 
+e2e-test-rollapp-upgrade-evm: clean-e2e
+	cd tests && go test -timeout=25m -race -v -run TestRollapp_EVM_Upgrade .
```

Comment:
we should change the name of test to `TestRollapp_Upgrade_EVM` to sync with another test @anhductn2001

Reply:
No. I intentionally left it like that so it can't be caught by the normal workflow, this test will only be triggered when there is an upgrade change